Makefile

```diff
@@ -1,6 +1,9 @@
 # Makefile assumes that direnv is active, or that pip/python on PATH
 # is what you want to use.
 
+# unstable protos are only included in local development and not part of release packages
+SENTRY_PROTOS_BUILD_UNSTABLE := 1
```

Comment:
I'm a little confused.  Won't this always activate?  We also use these make targets in CI so... https://github.com/getsentry/sentry-protos/blob/main/.github/workflows/build.yml#L23

Maybe I'm misremembering makefile precedence, but environment variables set in this way override ones provided by command line or environment?  So this would be 100%

Reply:
The `package-py` target sets the environment variable value when it runs `build-py`. Environment variables also overwrite the default values defined in `Makefile`